src/reduceWhile.js

```diff
@@ -0,0 +1,36 @@
+var _curryN = require('./internal/_curryN');
+var _reduce = require('./internal/_reduce');
+var _reduced = require('./internal/_reduced');
+
+
+/**
+ * Like `reduce`, `reduceWhile` returns a single item by iterating through
+ * the list, successively calling the iterator function. `reduceWhile` also
+ * takes a predicate that is evaluated at each step. If the predicate returns
```

Comment:
The word "before" is missing here, somewhere. It should be clear that the predicate is running before the reducer (and by extension, that the accumulator may be returned as-is).